api/envoy/extensions/outlier_detection_monitors/consecutive_errors/v3/consecutive_errors.proto

```diff
@@ -15,20 +15,17 @@ option java_multiple_files = true;
 option go_package = "github.com/envoyproxy/go-control-plane/envoy/extensions/outlier_detection_monitors/consecutive_errors/v3;consecutive_errorsv3";
 option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
+// [#protodoc-title: Consecutive errors monitor]
+// [#extension: envoy.outlier_detection_monitors.consecutive_errors]
+
 // Monitor which counts consecutive errors.
 // If number of consecutive errors exceeds the threshold, monitor will report that the host
 // is unhealthy.
-// [#not-implemented-hide:]
 message ConsecutiveErrors {
-  // Monitor name.
-  string name = 1;
+  // Base monitor config. Defines settings shared by all types of monitors.
+  common.v3.MonitorCommonSettings common_settings = 1 [(validate.rules).message = {required: true}];
```

Comment:
Prefer `common_config`. will we expect there is more shared config in future? If not, then use the enforcing here is fine.